### ISSUE TITLE
Is there a way to specify 'complex' markers

### ISSUE THREAD

**Issue by emayssat-ms** [philpep/testinfra#79]
When I run testinfra, I specify the test markers on the command line to run only a subsets of the tests.

Now I used to have --platform aws 
as a  command line option
and use pytest,mark,platform('aws') as a marker.

Now I are passing a configuration file to testinfra and therefore not specifying the --platform option anymore (since its value is in the config file).

Now, when I run testinfra without markers the AWS tests are run, when I use markers the test aren't run!
Is there a way to specify "platform('aws')" in the marker logic. 
It seems that markers should just be simple words to works 

Example command line:
AWS_PROFILE=3742 AWS_REGION=us-west-1 testinfra --capture=fd --connection=paramiko --sudo --hosts=ubuntu@54.67.57.119  -m "common or kafka or zookeeper" --maxfail 998 --tb=long --configuration-file configs/15_at_pre_deployment.yml test_suite/

In short, I would like to add platform('aws') in the -m option.
Is there a way to do so?

Thanks,
